js/models/conversations.js

```diff
@@ -273,7 +273,8 @@
       const recipientId = this.isPrivate() ? this.id : null;
       const groupNumbers = this.getRecipients();
 
-      const sendOptions = this.getSendOptions();
+      let sendOptions = this.getSendOptions();
+      sendOptions.messageType = "typing";
```

Comment:
I think our linter prefers single-quotes.
Have you tried running `yarn lint` before pushing?

Reply:
Yeah, I forgot that, and then noticed that travis complained :(

Reply:
updated